NuRadioReco/framework/base_station.py

```diff
@@ -174,18 +174,27 @@ def get_electric_fields_for_channels(self, channel_ids=None, ray_path_type=None)
                     yield e_field
                 elif ray_path_type == e_field.get_parameter(parameters.electricFieldParameters.ray_path_type):
                     yield e_field
+
     def is_neutrino(self):
+        if not self._particle_type == 'nu' and not self._particle_type == 'cr':
+            logger.error("Stations particle type was not properly set: \"{}\"".format(self._particle_type))
+            raise ValueError("Stations particle type was not properly set: \"{}\"".format(self._particle_type))
+
         return self._particle_type == 'nu'
-        
+
     def is_cosmic_ray(self):
+        if not self._particle_type == 'nu' and not self._particle_type == 'cr':
```

Comment:
I think it would be better to just check for particle_type == '' instead here. We just want to make sure it was initialized, but there may be other particle types that make sense.

Reply:
Agree, I can change that.